src/coreComponents/physicsSolvers/multiphysics/CoupledReservoirAndWellKernels.hpp

```diff
@@ -0,0 +1,592 @@
+/*
```

Comment:
The changes here don't seem well related...or thermal related. I missed it? Can you explain?

Reply:
Originally the well-res coupling derivatives were not in kernel classes but assembled in CompositionalMultiphaseReservoirAndWells.cpp.  With the addition of thermal ,kernels were added for iso + thermal and are in CoupledReservoirAndWellKernels.hpp